src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -201,6 +201,12 @@ internal void BuildAndLogConsoleLog(string msg, LogLevel level)
             {
                 _consoleLogSource?.Log(consoleLog);
             }
+
+            if (WorkerProcessUtilities.IsToolingConsoleJsonLogEntry(msg))
+            {
+                // log with the message prefix as coretools expects it.
+                _workerProcessLogger?.Log(level, msg);
+            }
```

Comment:
Do the same thing that the other usage of this did -- I'm not sure if that Log category is required, but let's not change it:

```
_toolingConsoleJsonLoggerLazy = new Lazy<ILogger>(() => loggerFactory.CreateLogger(WorkerConstants.ToolingConsoleLogCategoryName), isThreadSafe: true);
```